spec/requests/api/v1/events/nearest_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe 'GET /v1/events/nearests?lat=&lon=&radius=' do
+  it 'returns the events closest to the lat and lon' do
+    near_event = create(:event, :museum)
+    farther_event = create(:event, :cambridge)
+    intrepid = build(:event, :intrepid)
+    lat = intrepid.lat
+    lon = intrepid.lon
+    radius = 5
+
+    get "/v1/events/nearests?lat=#{lat}&lon=#{lon}&radius=#{radius}"
```

Comment:
You can use the rails path helpers here. Something like  `nearests_v1_events_path(lat: lat, lon: lon, radius: radius)`